src/main/java/me/sargunvohra/mcmods/autoconfig1u/serializer/GsonConfigSerializer.java

```diff
@@ -1,16 +1,16 @@
 package me.sargunvohra.mcmods.autoconfig1u.serializer;
 
-import com.google.gson.Gson;
-import com.google.gson.GsonBuilder;
-import com.google.gson.JsonParseException;
+import com.google.gson.*;
 import me.sargunvohra.mcmods.autoconfig1u.ConfigData;
```

Comment:
our static analysis engine detects that your pull request includes change from an explicit import to a wildcard import. According to the best practice (http://www.javadude.com/articles/importondemandisevil.html), import star (import on demand) is a bad practice, wildcard import can sometimes cause name conflicts and ambiguities.

Reply:
wait no I am concern how did this get here, ignore this

Reply:
Oh. I mean I fixed it in the latest commit but OK